src/Silverback.Integration.MQTT/Messaging/Configuration/Mqtt/MqttClientConfigBuilder.cs

```diff
@@ -350,12 +353,36 @@ public IMqttClientConfigBuilder EnableTls(Action<MqttClientOptionsBuilderTlsPara
             return this;
         }
 
+        /// <inheritdoc />
+        public IMqttClientConfigBuilder OnConnected(Action<MqttClientConfig> onConnected)
```

Comment:
Similar to the previous comment and to be in line with Kafka, I'd except these methods in the `IMqttConsumerEndpointBuilder`/`IMqttProducerEndpointBuilder` (see also https://github.com/BEagle1984/silverback/blob/master/src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/IKafkaConsumerEndpointBuilder.cs and https://github.com/BEagle1984/silverback/blob/master/src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/IKafkaProducerEndpointBuilder.cs).